suite/tests/CMakeLists.txt

```diff
@@ -3729,6 +3732,8 @@ if (BUILD_SAMPLES)
         endif ()
       elseif (sample STREQUAL "callstack")
         set(sample.${sample}_expectbase "sample.callstack")
+        torunonly_ci(sample.${sample} common.call_to_malloc
```

Comment:
So this overrides the prior call to `torunonly_ci(sample.${sample} ` at L3664. I see that's done by other conditional blocks above also, so maybe it works fine. Add a comment at L3664 that we override the test app in some cases. Or maybe it will be cleaner to move L3664 into the conditional blocks that actually do not override it later.

Reply:
Correct. It seems to work fine.
Added comment. All `if` cases end up overwriting it at some nesting level, so we'd end up with a lot of `else` cases with the common `torunonly_ci()`. I think it's more readable this way, where we only overwrite for the special cases.